qlib/linux_def.rs

```diff
@@ -2256,6 +2259,22 @@ impl PermMask {
 #[derive(Debug, Clone, Copy)]
 pub struct Flags(pub i32);
 
+#[cfg(target_arch = "x86_64")]
+impl Flags {
+    pub const O_DIRECT: i32 = 0o00040000; //0x00004000;
+    pub const O_LARGEFILE: i32 = 0o00100000; //0x00008000;
+    pub const O_DIRECTORY: i32 = 0o00200000; //0x00010000;
+    pub const O_NOFOLLOW: i32 = 0o00400000; //0x00020000;
+}
+
+#[cfg(target_arch = "aarch64")]
+impl Flags {
```

Comment:
Maybe open a new PR for this so that we can merge it already?
@mhomidi Do you want to add your signoff on it?!